spec/models/keyword_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Keyword, type: :model do
+  describe 'Validations' do
+    it { is_expected.to validate_presence_of(:name) }
```

Comment:
Usually if a validation can be covered by shoulda matchers, we dont need to test it manually.